feature-governance-impl/src/main/java/io/novafoundation/nova/feature_governance_impl/domain/referendum/details/call/treasury/TreasurySpendAdapter.kt

```diff
@@ -15,7 +15,8 @@ class TreasurySpendAdapter : ReferendumCallAdapter {
         call: GenericCall.Instance,
         context: ReferendumCallParseContext
     ): ReferendumCall? {
-        if (!call.instanceOf(Modules.TREASURY, "spend")) return null
+        // TODO: spend call is using MultiLocation now instead of MultiAddress so binding throws an exception
```

Comment:
Is this a problem that spend call now throws an exception? We want still display such referendums but without amount?